From 8aa7a039b6b23f7e9fdb9a459e975f216dd8fdba Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ha=C3=AFkel=20Gu=C3=A9mar?= <hguemar@fedoraproject.org>
Date: Sat, 13 Sep 2014 14:03:04 +0200
Subject: [PATCH 5/7] redis 2.8.18: sentinel configuration file fix

---
 sentinel.conf | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/sentinel.conf b/sentinel.conf
index 0e1b266..428e7b6 100644
--- a/sentinel.conf
+++ b/sentinel.conf
@@ -130,6 +130,12 @@ sentinel parallel-syncs mymaster 1
 # Default is 3 minutes.
 sentinel failover-timeout mymaster 180000
 
+#
+# Specify the log file name. Also the empty string can be used to force
+# Redis to log on the standard output. Note that if you use standard
+# output for logging but daemonize, logs will be sent to /dev/null
+logfile /var/log/redis/sentinel.log
+
 # SCRIPTS EXECUTION
 #
 # sentinel notification-script and sentinel reconfig-script are used in order
-- 
2.9.3

